docs/src/reference/gremlin-applications.asciidoc

```diff
@@ -732,7 +732,10 @@ The following table describes the various configuration options for the Gremlin
 |connectionPool.keepAliveInterval |Length of time in milliseconds to wait on an idle connection before sending a keep-alive request. Set to zero to disable this feature. |1800000
 |connectionPool.keyCertChainFile |The X.509 certificate chain file in PEM format. |_none_
```

Comment:
since some of these settings are deprecated, I think that we should just remove them from the documentation. we usually don't document deprecated concepts.